src/js/components/getWeather.js

```diff
@@ -0,0 +1,29 @@
+import Time from '../helpers/convertTime';
+import * as el from './elements';
+
+const getWeatherData = async (city) => {
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${el.key}&units=metric`;
+  const response = await fetch(url)
+    .then((response) => response.json())
+    .catch((error) => alert(error));
```

Comment:
- [ ] Do not use Window.alert() or Window.confirm(). 
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png)

Reply:
@DcRonan 
Opps, it obvious you added exactly what you saw in the link. 🤦 .
Sorry if I have mislead you.
I meant you could use `.catch`
```
.catch((error) => {
  console.error(error);
});
```
should work.
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch](url)

Reply:
Hi @bellom, not your fault. I have already sent it off again and have done pretty much what you have suggested, thanks.